worlds/bk_sudoku/docs/fr_Sudoku.md

```diff
@@ -0,0 +1,13 @@
+# Bk Sudoku
+
+## Qu'est-ce que ce jeu ?
+
+BK Sudoku n'est pas un jeu typique d'Archipelago ; il s'agit plutôt d'un client générique de Sudoku qui peut se connecter à n'importe quel multi-monde existant. Une fois connecté, vous pouvez jouer au Sudoku pour débloquer des indices aléatoires pour votre jeu. Bien que lent, il vous donnera quelque chose à faire lorsque vous n'arrivez pas à atteindre les vérifications dans votre jeu.
+
+## Quels sont les indices débloqués ?
+
+Après avoir terminé un puzzle Sudoku, le jeu débloquera un indice aléatoire pour un emplacement non trouvé dans la partie à laquelle vous êtes connecté.
```

Comment:
"puzzle" should be translated if possible. Suggestion: "(...) terminé une grille de Sudoku"

I believe the term "slot" could be replaced, and better understood, with "word". Thus, I suggest changing "la partie à laquelle" for "le monde que".

The part "débloquera un indice aléatoire pour un emplacement non trouvé" feels incorrect. It is not the indice that is random but the location hinted. Suggesting to replace the sentence with the following (assuming previous suggestion are accepted) "Après avoir terminé une grille de Sudoku, le jeu va choisir, de façon aléatoire, un des endroits non trouvé dans le monde que vous êtes connecté et indiquer l'objet qui s'y trouve.